xml/System.Text.Json/JsonException.xml

```diff
@@ -97,9 +96,9 @@
         <Parameter Name="innerException" Type="System.Exception" />
       </Parameters>
       <Docs>
-        <param name="message">To be added.</param>
-        <param name="innerException">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="message">The context specific error message.</param>
+        <param name="innerException">The exception that caused the current exception.</param>
+        <summary>Creates a new exception object to relay error information to the user.</summary>
```

Comment:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Text.Json.JsonException" /> class. with a specified error message and a reference to the inner exception that is the cause of this exception.</summary>
```

Reply:
I made a minor edit to your suggestion:
 . -> ,